### PR TITLE
Adding new option to docs: org and dept code

### PR DESCRIPTION
**Pull request by jteheran**

### CODE REVIEW

**Comment by DrDaveD** [comment]
This depends on the release of osg-pki-tools-3.2.3 in [SOFTWARE-3698](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3698)

File: docs/security/host-certs.md [hunk 1]
```diff
@@ -142,7 +142,10 @@ If you are not already a Registration Authority (RA) for your institution, you m
             user@host $ openssl pkcs12 -in incommon_file.p12 -nocerts -out ~/path_to_dir/incommon_user_key.pem
             user@host $ openssl pkcs12 -in incommon_file.p12 -nokeys -out ~/path_to_dir/incommon_user_cert.pem
 
-<a name="osg-incommon-cert-request"></a>
```
Comment (brianhlin) [inline docs/security/host-certs.md]: Please add this anchor back, it's used by a link above

File: docs/security/host-certs.md [hunk 2]
```diff
@@ -152,7 +155,7 @@ Once you have RA privileges, you may request, approve, and retrieve host certifi
         user@host $ osg-incommoncert-request --username <INCOMMON_LOGIN> \
                     --cert ~/path_to_dir/incommon_user_cert.pem \
                     --pkey ~/path_to_dir/incommon_user_key.pem \ 
-                    --hostname <HOSTNAME>
+                    --hostname <HOSTNAME> [--orgcode <ORG,DEPT>]
```
Comment (brianhlin) [inline docs/security/host-certs.md]: Could we put the `--orgcode` on a newline to mimic the formatting of the other options?

File: docs/security/host-certs.md [hunk 3]
```diff
@@ -162,15 +165,16 @@ Once you have RA privileges, you may request, approve, and retrieve host certifi
                     --pkey ~/path_to_dir/incommon_user_key.pem \
                     --hostname <HOSTNAME> \
                     --altname <ALTNAME> \
-                    --altname <ALTNAME2>
+                    --altname <ALTNAME2> [--orgcode <ORG,DEPT>]
```
Comment (brianhlin) [inline docs/security/host-certs.md]: Could we put the `--orgcode` on a newline to mimic the formatting of the other options?

**Comment by sandersjd8** [comment]
I got it to work when I used the numeric codes rather than the test equivalents. I only figured this out when I read the man page and saw the example. Could you please note that these are numeric rather than test in the -h? I wish there was a better way to look up the numeric codes than the API since I can only get the department code since I am one level down than our university's InCommon admin. Thanks for all the help.

Best regards,

David Sanders

**Comment by timtheisen** [comment]
@jteheran : Can we get these review comments addressed? Today is release day.

**Comment by jteheran** [comment]
Hi,
I'm off today. Sorry I did not have the chance to review these comments
yesterday.

--
Jeny

**Comment by brianhlin** [comment]
I've addressed the comments directly. @timtheisen you can release this whenever you're ready

**Pull request merged**